--- mbeditviz_callbacks.c	2022-11-07 02:55:13.000000000 -0400
+++ /home/epatton/MB-System-5.7.9beta44/src/mbeditviz/mbeditviz_callbacks.c	2022-10-21 14:50:14.324710831 -0300
@@ -1017,16 +1017,16 @@
     mbv_secondary_colortable_mode = MBV_COLORTABLE_NORMAL;
     mbv_secondary_colortable_min = mbev_grid.smin;
     mbv_secondary_colortable_max = mbev_grid.smax;
-    mbv_exageration = 1.0;
+    mbv_exageration = 5.0;
     mbv_modelelevation3d = 90.0;
     mbv_modelazimuth3d = 0.0;
     mbv_viewelevation3d = 90.0;
     mbv_viewazimuth3d = 0.0;
-    mbv_illuminate_magnitude = 1.0;
-    mbv_illuminate_elevation = 5.0;
+    mbv_illuminate_magnitude = 5.0;
+    mbv_illuminate_elevation = 15.0;
     mbv_illuminate_azimuth = 90.0;
-    mbv_slope_magnitude = 1.0;
-    mbv_overlay_shade_magnitude = 1.0;
+    mbv_slope_magnitude = 5.0;
+    mbv_overlay_shade_magnitude = 5.0;
     mbv_overlay_shade_center = 0.0;
     mbv_overlay_shade_mode = MBV_COLORTABLE_NORMAL;
     mbv_contour_interval = pow(10.0, floor(log10(mbev_grid.max - mbev_grid.min)) - 1.0);
